Dockerfile

```diff
@@ -1,52 +1,75 @@
-FROM php:7.1.15
-MAINTAINER Miroslav Cillik <miro@keboola.com>
+FROM php:7.3
+
 ENV DEBIAN_FRONTEND noninteractive
+ARG SNOWFLAKE_ODBC_VERSION=2.16.10
```

Comment:
tady bych dal buď stejnou verzi jako byla v .travis.yml `2.16.6` a nebo latest `2.16.11` nebo to má nějaký důvod proč latest-1?

Reply:
IMHO 2.16.11 vyšla někdy dneska nebo včera :D Dal jsem .11 ;)